packages/dds/merge-tree/src/snapshotV1.ts

```diff
@@ -253,7 +254,8 @@ export class SnapshotV1 {
         options: Properties.PropertySet,
         serializer?: IFluidSerializer,
     ): Promise<MergeTreeChunkV1> {
-        const chunkAsString: string = await storage.read(path);
+        const blob = await storage.readBlob(path);
+        const chunkAsString = blobToString(blob);
         return SnapshotV1.processChunk(path, chunkAsString, logger, options, serializer);
```

Comment:
If you go with my suggestion of using deserializeBlob() in more places, it would be great in cases like this to see if we can move seriazlier.parse() call out to this level from SnapshotV1.processChunk(), thus making processChunk simpler (does not need serializer), but also have more code reuse.